src/pages/my_page/user_profile/MyProjectRegistrations.tsx

```diff
@@ -56,6 +56,12 @@ export const MyProjectRegistrations = ({
       return new Date(a.startDate).getTime() - new Date(b.startDate).getTime();
     });
 
+  const projectsToShow = filteredProjects.slice(rowsPerPage * page, rowsPerPage * (page + 1));
+
+  if (page > projectsToShow.length * rowsPerPage) {
+    setPage(page - 1);
+  }
```

Comment:
I retrospekt tror jeg vi kan bruke `useEffect` til dette i stedet. Vil si det er naturlig å hoppe til første side hver gang man endrer filtrering, da man ellers vil rikisere å miste noen treff fordi det dukker opp på en tidligere side

```suggestion
  useEffect(() => {
    setPage(0);
  }, [selectedOngoing, selectedNotStarted, selectedConcluded]);
```

Reply:
Da får vi en liten feil i konsollen siden page ikke stemmer overens med projectsToShow.length en bitteliten periode. 
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/fdeb2cbd-c14d-4cc4-af08-21b4fdd30879)


Men det kan vi eventuelt løse ved å legge til en fallback (likt som du har/hadde under her egentlig
```
const validPage = page < Math.ceil(filteredProjects.length / rowsPerPage) ? page : 0;

<TablePagination
  page={validPage}
```